frontend/src/views/cancellations.js

```diff
@@ -5,34 +5,59 @@ import airplane from "../images/airplane.png";
 import { Button } from "../components/ui";
 import ValueCancelation from "../components/value-cancelation";
 import { useRound } from "../selectors";
+import { navigate } from "@reach/router";
+import Header from "../components/header";
+import { useSetCurrentGame } from "../action-hooks";
 
 function FlightDetails() {
   // TODO: WIP animation of values
   const game = useRound();
-  const seatsStatus = game.cancellations - game.overbookingNumber;
-  const totalRevenue = game.totalRevenue / 1000;
-  const [cancellation, setCancellations] = React.useState(game.cancellations);
-  const [overbookingCost, setOverbookingCost] = React.useState(
-    seatsStatus < 0 ? (seatsStatus * game.overbookingCost) / -1000 : 0
+  const {
+    cancellations,
+    overbookingNumber,
+    totalRevenue,
+    overbookingCost,
+    underageCost
+  } = game.games[game.currentGame];
+  const seatsStatus = cancellations - overbookingNumber;
+  const currentTotalRevenue = totalRevenue / 1000;
+  const [cancellation, setCancellations] = React.useState(cancellations);
+  const [currentOverbookingCost, setOverbookingCost] = React.useState(
+    seatsStatus < 0 ? (seatsStatus * overbookingCost) / -1000 : 0
   );
-  const [underageCost, setUnderageCost] = React.useState(
-    seatsStatus > 0 ? (seatsStatus * game.underageCost) / 1000 : 0
+  const [currentUnderageCost, setUnderageCost] = React.useState(
+    seatsStatus > 0 ? (seatsStatus * underageCost) / 1000 : 0
   );
   const [netRevenue, setNetRevenue] = React.useState(
-    totalRevenue - overbookingCost - underageCost
+    currentTotalRevenue - currentOverbookingCost - currentUnderageCost
   );
 
+  const setCurrentGame = useSetCurrentGame();
+
   React.useEffect(() => {}, [cancellation]);
 
+  function changeCurrentGame(event) {
+    event.preventDefault();
+    setCurrentGame(game.currentGame + 1);
+    navigate("/");
+  }
+
+  function goToScore(event) {
+    event.preventDefault();
```

Comment:
I think that the button that trigger this function don't have any events, so the `event.preventDefault` would not be necessary.